samples/samples-js/GetProductNamesView/index.js

```diff
@@ -0,0 +1,12 @@
+module.exports = async function (context, req, products) {
+    context.log('JavaScript HTTP trigger function processed a request.');
+    if (products.length === 0) {
```

Comment:
Clean up, we don't need this logging stuff here (and if we decide to add later we should add it to all languages)

(do this for all the other functions as well)